pokepro/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./card.css";
+type cardProps={
+  name:string;
+  imgURL:string;
+  prominentColor?: string;
+  ability?:string;
+}
+function Card({name,imgURL,prominentColor ,ability,}:cardProps) {
```

Comment:
Can we use the `React.FC` type here
Let's also name the component to a specific name, like PokemonCard or Something.

```suggestion
const Card: React.FC<CardProps> = ({name, imgURL, prominentColor ,ability}) => {
```